x/accounts/protoutils.go

```diff
@@ -0,0 +1,117 @@
+package accounts
+
+import (
+	"time"
+
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	txv1beta1 "cosmossdk.io/api/cosmos/tx/v1beta1"
+	"cosmossdk.io/x/accounts/internal/implementation"
+
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+)
+
+func protoV2TxToProtoV1(t *txv1beta1.Tx) *tx.Tx {
```

Comment:
Why don't we just marshal and unmarshal? This is probably marginally faster but less safe.